rancher/v1.5/en/cattle/scheduling/index.md

```diff
@@ -8,25 +8,24 @@ lang: en
 ## Scheduling Services
 ---
 
-Rancher supports container scheduling policies that are modeled closely after Docker Swarm.  They include scheduling based on:
+When Rancher schedules containers on hosts, there are default scheduling policies that it adheres to. These policies include:
 
-* port conflicts
-* shared volumes
-* host tagging
-* shared network stack: `--net=container:dependency`
-* strict and soft affinity/anti-affinity rules by using both env var (Swarm) and labels (Rancher)
+* Spreading containers across hosts in `active` state
+* Avoiding port conflicts
 
```

Comment:
These two rules are built in external scheduler, so if the external scheduler is off cattle lost the ability to provide those two scheduling rule. So i am not sure if it is proper to call them the default scheduling policy.